docker-compose.yml

```diff
@@ -122,6 +122,45 @@ services:
     labels:
       logs: "promtail"
 
+### Web Client
+  hyperswitch-web:
+    ports:
+      - "9050:9050"
+      - "9060:9060"
+      - "5252:5252"
+    build:
+      context: ./docker
+      dockerfile: web.Dockerfile
+    networks:
+      - router_net
+    env_file:
+      - ./.env
+    environment:
+      - HYPERSWITCH_PUBLISHABLE_KEY=$HYPERSWITCH_PUBLISHABLE_KEY
+      - HYPERSWITCH_SECRET_KEY=$HYPERSWITCH_SECRET_KEY
```

Comment:
@prasad89 is it necessary to pass in a publishable & secret key?
what would the behaviour be if these env variables are not set

@SanchithHegde what do you feel here about the requirement of a publishable key & secret key as part of starting up our stack?

should we move the demo app to a separate docker service or exclude it from the requirements?

Reply:
UI will not be loaded without that.

Reply:
Ok,
as long as `http://localhost:9050/HyperLoader.js`(the sdk) loads without the publishable key, this should be fine then,
thanks for clearing that up